packages/blockfrost/src/tsconfig.json

```diff
@@ -1,11 +1,15 @@
 {
   "extends": "../../../tsconfig.json",
   "compilerOptions": {
-    "outDir": "../dist/",
+    "outDir": "../dist/cjs",
     "rootDir": "."
   },
   "references": [
-    { "path": "../../util/src" },
-    { "path": "../../core/src" }
+    {
```

Comment:
Any ideea why this is being formatted like this?

Reply:
My vscode autoformats to this. I don't seem to have any extra formatting plugins so it must be from eslint :thinking:

Reply:
Agree, formatting should differ based on who makes the changes, but we have ADP-1803 to work on this